### DISCUSSION TITLE
Allow marking any reply as an answer in Discussions with answers enabled.

### DISCUSSION THREAD

**Discussion by NSExceptional** [community/community#3892] [Discussions]
Currently, you can only mark top-level replies as an answer. This is not very useful, because usually a Q&A thread will result in some back-and-forth before an answer is reached.

**Comment by Andre601** [comment]
I can agree to this.
More often than not do some answer happen in replies to a top-level response, which then can't be selected unless the author or someone else creates a separate response to be selectable.

This has and still is a major downside in StackOverflow (Which GitHub obviously took inspiration from in certain sections) which I dislike a lot, so GitHub should try to be better in that regard.
